ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -78,20 +73,23 @@ func (scbuilder *SiteConfigBuilder) getClusterCRs(clusterId int, siteConfigTemp
 			cluster := siteConfigTemp.Spec.Clusters[clusterId]
 			dataMap, err := scbuilder.getExtraManifest(dataMap, cluster)
 			if err != nil {
-				return clusterCRs, err
+				log.Printf("warning could not read extra-manifest %s.%s %s\n", cluster.ClusterName, cluster.ExtraManifestPath, err)
 			}
 
-			// FIXME: Assuming 1 node for SNO deployment needs to be changed for RWN deployment
-			if len(siteConfigTemp.Spec.Clusters[clusterId].Nodes) > 0 {
+			// Adding workload partitions MC only for SNO clusters.
+			if siteConfigTemp.Spec.Clusters[clusterId].ClusterType == SNO &&
+				len(siteConfigTemp.Spec.Clusters[clusterId].Nodes) > 0 {
 				cpuSet := siteConfigTemp.Spec.Clusters[clusterId].Nodes[0].Cpuset
 				if cpuSet != "" {
 					k, v, err := scbuilder.getWorkloadManifest(cpuSet)
 					if err != nil {
-						return clusterCRs, err
+						log.Printf("warning could not read WorkloadManifest %s %s\n", cluster.ClusterName, err)
```

Comment:
If this fails we will end up deploying a cluster without workload partitioning which is not good. We should consider it a failure if any of our baseline/required CRs aren't created for the cluster.

Reply:
okay, so it will follow the same logic and return empty configMap then it will break in the outer caller return.